packages/ui/src/types/authenticator/attributes.ts

```diff
@@ -0,0 +1,63 @@
+// https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-settings-attributes.html
+export const signUpFieldsWithDefault = [
```

Comment:
Could this be broken out further into a signUpFields file? Half this file is just SignUp related types.

Reply:
Good point. To be fair they're misnomers --  these fields are applicable to other screens like `ForgotNewPassword` and not just `SignUp` (#1155, #1176). 

I'm doing a form refactor today, so I'll re-address it there.